collective/pdfpeek/browser/utils.py

```diff
@@ -29,6 +31,9 @@ class pdfpeekUtilView(BrowserView):
     def process_conversion_queue(self):
         """process the queue.
         """
+        token = createToken()
+        self.request.set('_authenticator', token)
+
```

Comment:
Wouldn't it be better to disable the csrf protection explicitly instead of adding a generated token to the request? In the end this still circumvents the csrf protection, which is ok, but then I would rather use the IDisableCSRFProtection as it's done in PR #5